packages/jaeger-ui-components/src/TraceTimelineViewer/TimelineRow.tsx

```diff
@@ -61,7 +61,7 @@ export function TimelineRowCell(props: TimelineRowCellProps) {
   const widthPercent = `${width * 100}%`;
   const mergedStyle = { ...style, flexBasis: widthPercent, maxWidth: widthPercent };
   return (
-    <div className={cx(ubRelative, className)} style={mergedStyle} {...(rest as any)}>
+    <div className={cx(ubRelative, className)} style={mergedStyle} {...(rest as any)} data-testid="TimelineRowCell">
```

Comment:
```suggestion
    <div className={cx(ubRelative, className)} style={mergedStyle} data-testid="TimelineRowCell" {...rest}>
```
It looks like by putting this testid after the rest props you're overriding `span-view` testid, which is assigned to the same component. In general we need to be careful which components we add testid to as it seems we assign different ids to the same components.